lib/ecto/discrete_range.ex

```diff
@@ -0,0 +1,311 @@
+defmodule Ecto.DiscreteRange do
+  @moduledoc """
+  Define useful functions to create an ecto.type for the postgres
```

Comment:
In this case it would be nice to quote to the ecto type module as ex_doc would autolink it's docs

Ie: 

```
Black black `Ecto.Type` food desu
```